<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Charts</class>
 <widget class="QWidget" name="Charts">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1100</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" rowstretch="10,80">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBoxSettings">
     <property name="title">
      <string>GroupBox</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="pushButtonStartMonitoring">
        <property name="text">
         <string>Rozpocznij monitorowanie</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonStopMonitoring">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Zako≈Ñcz monitorowanie</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QCustomPlot" name="customPlot" native="true"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot/qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
